elasticsearch/sgconfig/sg_roles.yml

```diff
@@ -35,6 +35,7 @@ sg_role_rsyslog:
 sg_role_curator:
   cluster:
     - CLUSTER_MONITOR
+    - MANAGE
```

Comment:
We can do this or create a `jaeger_curator` with this role.

Reply:
I don't see a reason why not to add this here. It seems to me that we would want our curator to be able to manage as well
@lukas-vlcek ?

Reply:
This give `MANAGE` rights to every index in the cluster.  Is this desired?

Reply:
just for Curator though, and it is tied to its SAN. If we want to have curator do rollover ourselves it would require this